external/test-runtime/optional.json

```diff
@@ -4,7 +4,9 @@
       "dependencies": {
         "Microsoft.DotNet.IBCMerge": "4.6.0-alpha-00001",
         "Microsoft.DotNet.UAP.TestTools": "1.0.2",
-        "TestILCNugetPackageForCoreFX": "1.0.0-beta-25304-01"
+        "TestILC.amd64ret": "1.0.0-beta-25305-00",
```

Comment:
Why do we still have this optional.json file? I thought we no longer use it. If we still have this dependency we should at least file a bug to switch it to PackageReference and then you can eliminate the need for the duplicated packages.